resources/azure-staticwebapps.svg

```diff
@@ -1,19 +1,17 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" viewBox="0 0 18 18">
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 18 18">
   <defs>
```

Comment:
Make sure to update the png used in the marketplace, too

https://github.com/microsoft/vscode-azurestaticwebapps/blob/main/resources/azure-staticwebapps.png